apps/marketing/src/components/(marketing)/confetti-screen.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+
+import React, { useEffect, useState } from 'react';
+
+import Confetti from 'react-confetti';
+import { createPortal } from 'react-dom';
+
+import { useIsMounted } from '@documenso/lib/client-only/hooks/use-is-mounted';
+import { useWindowSize } from '@documenso/lib/client-only/hooks/use-window-size';
+
+export default function ConfettiScreen(
+  props: React.ComponentPropsWithoutRef<typeof Confetti> & { duration?: number },
+) {
+  const { width, height } = useWindowSize();
+  const isMounted = useIsMounted();
+
+  const [numberOfPieces, setNumberOfPieces] = useState(props.numberOfPieces ?? 200);
```

Comment:
When doing things like this we should destructure the props, so instead you would do:

```
export default function ConfettiScreen({ numberOfPieces = 200, ...props }: ....) {
   ....
   const [numberOfPieces, setNumberOfPieces] = useState(numberOfPieces);
}
```